ios/engine/KMEI/KeymanEngine/Classes/Manager.swift

```diff
@@ -181,9 +181,12 @@ public class Manager: NSObject, HTTPDownloadDelegate, UIGestureRecognizerDelegat
     keymanWeb.view.addGestureRecognizer(hold)
 
     reachability = Reachability(hostName: keymanHostName)
```

Comment:
Should `reachability` only be initialized for non-system keyboards?

Reply:
I'm pretty sure keyboard updates are only done from within the app, and never from the system keyboard.  Is it the same for the Android app?

Reply:
For the Android app, keyboard updates can be downloaded from the system keyboard.
(I tested with an app the uses european2-1.0.js as the default keyboard)